src/Microsoft.AspNet.Mvc.Core/ActionResults/FileStreamResult.cs

```diff
@@ -39,7 +41,7 @@ public FileStreamResult([NotNull] Stream fileStream)
         /// </summary>
         /// <param name="fileStream">The stream with the file.</param>
         /// <param name="contentType">The Content-Type header of the response.</param>
-        public FileStreamResult([NotNull] Stream fileStream, string contentType)
+        public FileStreamResult([NotNull] Stream fileStream, MediaTypeHeaderValue contentType)
```

Comment:
I think we are not...this question is applicable to FilePathResult too which uses the following SendFileFeature in some situations.../cc: @Tratcher 

https://github.com/aspnet/Mvc/blob/a9feed4de1eb45bad0750bc868900b20cf842cc7/test/WebSites/FilesWebSite/SendFileMiddleware.cs#L50